web/containers/admin/GenericApplicantContainer/index.js

```diff
@@ -460,8 +460,28 @@ class GenericApplicantContainer extends React.Component {
       }
     });
     const filteredApplicants = this.filterApplicants(checkedApplicants);
-    const fields = ['firstName', 'lastName', 'email', 'id', 'tags'];
+    const fields = ['firstName', 'lastName', 'email', 'id', 'tags.accepted', 'tags.abc'];
```

Comment:
Can we set fields to an empty array?

Reply:
we could, i think it's better if we keep firstName, lastName, email, and id so it keeps it nicely ordered, what do you think?

Reply:
That's a good point, I'm fine with keeping `firstName`, `lastName`, `email`, and `id`, but can we get rid of `tags.accepted`, `tags.abc`?